en/full-node.md

```diff
@@ -0,0 +1,362 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+layout: base
+lang: en
+id: full-node
+title: "Running A Full Node - Bitcoin"
+---
+
+{% assign moreHelp="If you need more help, please ask in one of Bitcoin's many [communities](/en/community), such as [Bitcoin StackExchange](https://bitcoin.stackexchange.com/), [BitcoinTalk technical support](https://bitcointalk.org/index.php?board=4.0), or the [#bitcoin](https://webchat.freenode.net/?channels=bitcoin&uio=d4) IRC chatroom on Freenode." %}
+
+# Running A Full Node
+
+<p class="summary">Learn how run a full node</p>
```

Comment:
Maybe making it a bit longer "Learn how run a full node on your computer or server." would be more readable since this string otherwise looks like another title between two titles?